### DISCUSSION TITLE
PingPong doesn't work for verbs provider on Windows

### DISCUSSION THREAD

**Discussion by mason1504** [ofiwg/libfabric#10617] [Q&A]
I'm having issues getting the pingpong example application to work for the verbs provider on Windows (using Mellanox CX5 adaptors). I'm running the applcation as follows:

Server: -p verbs -s <local_ip_address> -v -e msg
Client: -p verbs -s <local_ip_address> <remote_ip_address> -v -e msg

Both client and server do not return from blocking reads on the event queue after fi_connect is called on the client side. Has anyone else seen this?

Thanks in advance.

**Comment by shefty** [comment]
Many of the fabtests will build and run on windows.  The windows CI runs these tests (e.g. msg_pingpong), but over the tcp provider.   (Verbs devices aren't available on appveyor.)  I don't think pingpong (from the libfabric build) is executed, but I don't know if there's a reason it isn't, other than it provides less test coverage than fabtests.
